gluon/tools.py

```diff
@@ -2214,7 +2214,19 @@ def define_tables(self, username=None, signature=None, enable_tokens=False,
             fake_migrate = db._fake_migrate
         settings = self.settings
         settings.enable_tokens = enable_tokens
-        super(Auth, self).define_tables(username, signature, migrate, fake_migrate)
+
+        if not self.signature:
+            self.define_signature()
+        if signature is True:
+            signature_list = [self.signature]
+        elif not signature:
+            signature_list = []
+        elif isinstance(signature, Table):
+            signature_list = [signature]
+        else:
+            signature_list = signature
+
+        super(Auth, self).define_tables(username, signature_list, migrate, fake_migrate)
```

Comment:
The problem is that Auth.define_tables also needs to have signature_list available if it wants to define more tables than AuthAPI. Would you prefer that AuthAPI.define_tables sets an auth instance variable with the signature list (e.g. self._table_signature_list)?